lexmapr/pipeline.py

```diff
@@ -187,6 +187,57 @@ def retainedPhrase(termList):
         returnedSetFinal = set(returnedSet)
     return returnedSetFinal
 
+def get_resource_dict(file_name, lower=False):
+    """Return dictionary containing resource data from a CSV file.
+
+    Arguments:
+        * file_name <class "str">: CSV file containing key-value
+            information on resources relevant to pipeline
+    Return values:
+        * class <"dict">: Contains key-value pairs from file_name
+            * key: class <"str">
+            * val: class <"str">
+    Restrictions:
+        * No information will be taken from the first row of file_name
+        * Unique keys must appear as the first term on unique rows
+        * Values should appear immediately after their corresponding
+            key and a comma
+            * Otherwise, an empty string value will be used
+    Optional arguments:
+        * lower <class "bool">: If set to True, all keys in the return
+            value are converted to lowercase
+    """
+    # Return value
+    ret = {}
+    # Open file_name
+    with open(resource_filename('lexmapr.resources', file_name)) as csvfile:
+        # Read file_name
+        file_contents = csv.reader(csvfile, delimiter=",")
+        # Flag indicating iteration beyond first row
+        first_row_seen = False
+        # Iterate across rows in file_contents
+        for row in file_contents:
+            # This is not the first row
+            if first_row_seen:
```

Comment:
Would it be more concise to call `next(csvfile)` once to advance past the first line instead of tracking this `first_row_seen` variable? As described here: https://stackoverflow.com/a/14674406